static/js/home/index.js

```diff
@@ -209,6 +209,9 @@ MEDUSA.home.index = function() {
                 },
                 indexer: function(itemElem) {
                     var indexer = $(itemElem).attr('data-indexer');
+                    if (indexer === undefined) {
```

Comment:
the indexer is undefined for the temporary poster that's shown for the show that's being added.